rules/integrations/cloud_defend/execution_interactive_shell_spawned_from_inside_a_container.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/04/26"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects when an interactive shell is spawned inside a running container. This could indicate an attacker's attempt to gain unauthorized access to the container."
```

Comment:
Edit: OR suspicious interactive shell

Reply:
meant unauthorized access to the host. Thinking of container breakout attempts here, will update the description